### PR TITLE
JAVA-1040: SimpleStatement parameters support in QueryLogger

### PR DESCRIPTION
**Pull request by tommystendahl**

### CODE REVIEW

File: driver-core/src/main/java/com/datastax/driver/core/QueryLogger.java [hunk 1]
```diff
@@ -746,6 +760,61 @@ protected String parameterValueAsString(ColumnDefinitions.Definition definition,
         return valueStr;
     }
 
+    protected int appendParameters(SimpleStatement statement, StringBuilder buffer, int remaining) {
```
Comment (tolbertam) [inline driver-core/src/main/java/com/datastax/driver/core/QueryLogger.java:763]: There is quite a bit of duplication of code between the appendParameters/parameterValueAsString methods for the SimpleStatement and BoundStatement implementations.  However, they are probably different enough to justify it being the way it is.

**Comment by tolbertam** [comment]
Looks good to me :+1:

File: driver-core/src/test/java/com/datastax/driver/core/QueryLoggerTest.java [hunk 2]
```diff
@@ -610,11 +723,32 @@ public void should_truncate_blob_parameter_when_max_length_exceeded() throws Exc
                 .contains("Query completed normally")
                 .contains(ipOfNode(1))
                 .contains("c_blob:0x0102" + TRUNCATED_OUTPUT)
-                .doesNotContain("123456");
+                .doesNotContain("0x010203");
```
Comment (adutra) [inline driver-core/src/test/java/com/datastax/driver/core/QueryLoggerTest.java:726]: Thanks for spotting this :)

File: driver-core/src/main/java/com/datastax/driver/core/QueryLogger.java (hunk 1 above)
Comment (olim7t) [inline driver-core/src/main/java/com/datastax/driver/core/QueryLogger.java:763]: Agreed

**Comment by olim7t** [comment]
Merged, thanks for your contribution!

**Pull request merged**